### PR TITLE
added regex to get_columns_names()

### PR DESCRIPTION
**Pull request by xdssio**
get_columns_namesa now expect regex pattern.
All other argumetns still apply. 

```
ds.get_column_names(regex='e*')
```

### CODE REVIEW

File: packages/vaex-core/vaex/dataset.py [hunk 1]
```diff
@@ -5863,3 +5879,4 @@ def values(self):
         If any of the columns contain masked arrays, the masks are ignored (i.e. the masked elements are returned as well).
         """
         return self.__array__()
+
```
Comment (maartenbreddels) [inline packages/vaex-core/vaex/dataset.py]: and no spaces :)

**Comment by maartenbreddels** [comment]
yeah, just check the 'changes' tab on github, or use gitk to see you don't introduce unneeded changes.

**Comment by xdssio** [comment]
What spaces? 


Jonathan Alexander
Consultant, Data Science
jonathan@xdss.io

**Comment by maartenbreddels** [comment]
On the bottom (see changes tab):
screen shot 2018-12-05 at 10 56 18

**Comment by maartenbreddels** [comment]
Almost feeling claustrofobic :)
Thanks!

**Pull request merged**